src/config/config.c

```diff
@@ -1452,6 +1452,8 @@ void readFTLconf(struct config *conf, const bool rewrite)
 	writeFTLtoml(true);
 	write_dnsmasq_config(conf, false, NULL);
 	write_custom_list();
+
+	return true;
```

Comment:
```suggestion
	return false;
```

When we reach this end, we failed to read the TOML and have actually just created one from `setupVars.conf` and friends